src/io/export_pov.cc

```diff
@@ -0,0 +1,114 @@
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ */
+
+#include "export.h"
+
+#include "PolySet.h"
+#include "PolySetUtils.h"
+
+
+void export_pov(const std::shared_ptr<const Geometry>& geom, std::ostream& output, const ExportInfo& exportInfo)
+{
+  auto ps = PolySetUtils::getGeometryAsPolySet(geom);
+  if (Feature::ExperimentalPredictibleOutput.is_enabled()) {
+    ps = createSortedPolySet(*ps);
+  }
+
+  output << "// Generated by OpenSCAD!\n";
+  output << "// Source file: " << exportInfo.sourceFileName << "\n\n";
+
+  output << "#version 3.7;\n";
+  output << "global_settings { assumed_gamma 1.0 }\n";
+  output << "#declare MATERIAL=finish { specular 0.5 roughness 0.001 reflection{0 0.63 fresnel} ambient 0 diffuse 0.6 conserve_energy }\n";
+  output << "#declare MATERIAL_INT=interior{ior 1.32}\n";
+
+  auto has_color = !ps->color_indices.empty();
+
+  for (size_t polygon_index=0; polygon_index<ps->indices.size(); polygon_index++) {
+    const auto &polygon = ps->indices[polygon_index];
+    output << "polygon { " << polygon.size() + 1 << ", \n";
+    for (size_t i=0; i<polygon.size(); i++) {
+      if (i)
+        output << ", ";
+      const auto & x = ps->vertices[polygon[i]].x();
+      const auto & y = ps->vertices[polygon[i]].y();
+      const auto & z = ps->vertices[polygon[i]].z();
+      output << "<" << x << ", " << y << ", " << z << ">";
+    }
+    output << ", <" << ps->vertices[polygon[0]].x() << ", " << ps->vertices[polygon[0]].y() << ", " << ps->vertices[polygon[0]].z() << ">";
+    float r = 1.0, g = 1.0, b = 1.0, a = 0.0;
+    if (has_color) {
+      auto color_index = ps->color_indices[polygon_index];
+      if (color_index >= 0) {
+        auto color = ps->colors[color_index];
+        r = float(color[0]);
+        g = float(color[1]);
+        b = float(color[2]);
+        a = 1.0 - color[3];
+      }
+    }
+    if (r == 0 && g == 0 && b == 0)  // work around for black objects in 5185
+      g = 1.0;
+    output << "\n";
+    output << "texture { pigment { color rgbf <" << r << ", " << g << ", " << b << ", " << a << "> } }\n";
+    output << "finish { MATERIAL } interior { MATERIAL_INT }\n";
+    output << "}\n";
+  }
+
+  BoundingBox bbox = geom->getBoundingBox();
+
+  auto & min_x = bbox.min().x();
+  auto & min_y = bbox.min().y();
+  auto & min_z = bbox.min().z();
+
+  auto & max_x = bbox.max().x();
+  auto & max_y = bbox.max().y();
+  auto & max_z = bbox.max().z();
+
+  double dx = max_x - min_x;
+  double dy = max_y - min_y;
+  double dz = max_z - min_z;
+
+  constexpr double move_away_factor = 2.;
+  std::vector<double> lx { min_x - dx * move_away_factor, 0, max_x + dx * move_away_factor };
+  std::vector<double> ly { min_y - dy * move_away_factor, 0, max_y + dy * move_away_factor };
+  std::vector<double> lz { min_z - dz * move_away_factor, 0, max_z + dz * move_away_factor };
+
```

Comment:
the zeroes here should probably be the center of the bounding box.
FYI: bbox.center().